src/Api.ts

```diff
@@ -218,6 +218,11 @@ namespace API {
         return new Challenge(data);
     }
 
+    export async function updateUser(ch: User | { id: number }) {
```

Comment:
variabele naam ch aanpassen naar iets wat lijkt op user. ch is namelijk oude afkorting van challenge